src/test/java/com/znsio/teswiz/screen/vodqa/VodqaScreen.java

```diff
@@ -57,16 +59,28 @@ public static VodqaScreen get() {
     public abstract VodqaScreen scrollDownInDynamicLayer(Direction direction);
 
     public abstract boolean isElementWithTextVisible(String elementText);
-  
+
     public abstract VodqaScreen scrollVerticallyByPercentage(int fromPercentHeight, int toPercentHeight, int percentWidth);
 
     public abstract VodqaScreen longPressOnElement();
 
     public abstract boolean isLongPressedPopupVisible();
-  
+
     public abstract DragAndDropScreen openDragAndDropScreen();
 
+    public abstract VodqaScreen navigateToPhotoView();
```

Comment:
can this be renamed to navigateToImageView from navigateToPhotoView?

Reply:
renamed to navigateToImageView